ms-windows/build-tools.php

```diff
@@ -0,0 +1,1561 @@
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MDd", $libdir . "/MDd", false, '/(\.lib|\.def)$/');
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MT", $libdir . "/MT", false, '/(\.lib|\.def)$/');
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MTd", $libdir . "/MTd", false, '/(\.lib|\.def)$/');
+
+			// OpenSSL.exe.
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MD/apps", $destdir . "/bin", false, '/(\.exe)$/');
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/static_MT/apps", $destdir . "/bin_static", false, '/(\.exe)$/');
+
+			// Configurations.
+			copy($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MD/apps/openssl.cnf", $destdir . "/bin/openssl.cfg");  // This is very old and is a long story.
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MD/apps", $destdir . "/bin/cnf", false, '/(\.cnf)$/');
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MD/apps", $destdir . "/conf", false, '/(\.cnf)$/');
+
+			// Perl scripts.
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MD/apps", $destdir . "/bin", false, '/(\.pl)$/');
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MD/tools", $destdir . "/tools", true);
+
+			// Test suites.
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MD/test", $destdir . "/tests", true);
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MD", $destdir . "/tests", false, '/(\.dll)$/');
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MD/fuzz", $destdir . "/tests/fuzz", true);
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MD", $destdir . "/tests/fuzz", false, '/(\.dll)$/');
+
+			// PEM files.
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MD/apps", $destdir . "/bin/PEM", false, '/(\.pem|\.srl)$/');
+			if (is_dir($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MD/apps/demoCA"))  CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MD/apps/demoCA", $destdir . "/bin/PEM/demoCA", true);
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/dll_MD/apps/demoSRP", $destdir . "/bin/PEM/demoSRP", true);
+
+			// Includes.
+			CopyFiles($pathmap["[[BUILD_OUTPUT_DIR]]"] . "/inc", $destdir . "/include", true);
+
+			// Text files.
+			@mkdir($destdir . "/text", 0775, true);
+			CopyTextFile($pathmap["[[SOURCE_ORIG_DIR]]"] . "/ACKNOWLEDGEMENTS", $destdir . "/text/acknowledgements.txt");
+			CopyTextFile($pathmap["[[SOURCE_ORIG_DIR]]"] . "/AUTHORS", $destdir . "/text/authors.txt");
+			CopyTextFile($pathmap["[[SOURCE_ORIG_DIR]]"] . "/CHANGES", $destdir . "/text/changes.txt");
+			CopyTextFile($pathmap["[[SOURCE_ORIG_DIR]]"] . "/FAQ", $destdir . "/text/faq.txt");
+			CopyTextFile($pathmap["[[SOURCE_ORIG_DIR]]"] . "/LICENSE", $destdir . "/text/license.txt");
+			CopyTextFile($pathmap["[[SOURCE_ORIG_DIR]]"] . "/NEWS", $destdir . "/text/news.txt");
+			CopyTextFile($pathmap["[[SOURCE_ORIG_DIR]]"] . "/README", $destdir . "/text/readme.txt");
+
+			CopyTextFile($pathmap["[[SOURCE_ORIG_DIR]]"] . "/LICENSE", $destdir . "/license.txt");
```

Comment:
This is mostly historical but also because it's Windows.  .txt will open up the file in Notepad by default.  Both .md and no file extension will ask the user what application they want to open up the file in and usually make bad recommendations.  On my machine, the first recommendation by Windows when opening `LICENSE` or `NEWS` is to open the file in Adobe Photoshop.